### ISSUE TITLE
verdi quick setup

### ISSUE THREAD

**Issue by SaraFio** [aiidateam/aiida-core#2912]
Ubuntu 16.04.4 LTS
Aiida core 1.0.0b3
```
(my_env) sara@sara-Inspiron-7348:~$ verdi quicksetup
Profile name [quicksetup]: sarafio
Email Address (identifies your data when sharing): safiore@iis.ee.ethz.ch
First Name: sara
Last Name: fiore
Institution: ethz  
Detected no known postgres setup, some information is needed to create the aiida database and grant 
aiida access to it. If you feel unsure about the following parameters, first check if postgresql is 
installed. If postgresql is not installed please exit and install it, then run verdi quicksetup again.
If postgresql is installed, please ask your system manager to provide you with the following parameters:
postgres host [localhost]: 
postgres port [5432]: 
template [template1]: 
postgres super user [postgres]: 

trying to access postgres..
postgres password of postgres []: 
postgres host [localhost]: ^CTraceback (most recent call last):
  File "/home/sara/.local/bin/verdi", line 11, in <module>
    sys.exit(run())
  File "/home/sara/.local/lib/python2.7/site-packages/aiida/cmdline/verdilib.py", line 1050, in run
    aiida.cmdline.verdilib.exec_from_cmdline(sys.argv)
  File "/home/sara/.local/lib/python2.7/site-packages/aiida/cmdline/verdilib.py", line 1035, in exec_from_cmdline
    CommandClass.run(*argv[command_position + 1:])
  File "/home/sara/.local/lib/python2.7/site-packages/aiida/cmdline/verdilib.py", line 622, in run
    quicksetup.invoke(ctx)
  File "/home/sara/.local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/sara/.local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/sara/.local/lib/python2.7/site-packages/click/decorators.py", line 27, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/home/sara/.local/lib/python2.7/site-packages/aiida/cmdline/verdilib.py", line 655, in quicksetup
    success = postgres.determine_setup()
  File "/home/sara/.local/lib/python2.7/site-packages/aiida/control/postgres.py", line 138, in determine_setup
    self._no_setup_detected()
  File "/home/sara/.local/lib/python2.7/site-packages/aiida/control/postgres.py", line 222, in _no_setup_detected
    self.dbinfo)
  File "/home/sara/.local/lib/python2.7/site-packages/aiida/control/postgres.py", line 252, in prompt_db_info
    'postgres host', default='localhost', type=str)
  File "/home/sara/.local/lib/python2.7/site-packages/click/termui.py", line 89, in prompt
    value = prompt_func(prompt)
  File "/home/sara/.local/lib/python2.7/site-packages/click/termui.py", line 80, in prompt_func
    raise Abort()
click.exceptions.Abort
```

**Comment by nathanielng** [comment]
Have you installed postgresql, which is one of the [prequisites](https://aiida-core.readthedocs.io/en/latest/install/prerequisites.html)? The setup script is asking for the name of the postgres super user / password that would have been set up in the steps mentioned [here](https://aiida-core.readthedocs.io/en/latest/install/installation.html)

**Comment by SaraFio** [comment]
Yes, I am sorry I forget to specify before that I installed and checked all the prerequisite required

**Comment by ltalirz** [comment]
This issue should be resolved by #3892 , which includes two fixes:

 * the default postgres setup on ubuntu 16.04 should be detected out of the box
 * the (broken) prompt loop for providing the connection details manually should be fixed
